src/commands/ring/create.ts

```diff
@@ -28,15 +27,16 @@ export const checkDependencies = (
 ): void => {
   const fileInfo: BedrockFileInfo = bedrockFileInfo(projectPath);
   if (fileInfo.exist === false) {
-    throw new Error(PROJECT_INIT_DEPENDENCY_ERROR_MESSAGE);
+    throw buildError(errorStatusCode.VALIDATION_ERR, "ring-err-dependency");
```

Comment:
i suggest we be more specific on error identifier `ring-err-dependency` -> `ring-create-cmd-err-dependency`. From what I understood during design review meeting are 1. We want more precise id and also given a id we can pin point where the error is thrown

Reply:
@dennisseah this error is used in all three commands though: create, delete, and set-default. What if the name is `ring-cmd-err-dependency` and I add a parameter {0} to indicate on which command it failed: "Error running {0}. Please run...."